MapleServer2/PacketHandlers/Game/GuildHandler.cs

```diff
@@ -541,6 +547,35 @@ private static void HandleListGuild(GameSession session, PacketReader packet)
         session.Send(GuildPacket.ListGuildUpdate(session.Player, toggle));
     }
 
+    private static void HandleGuildMail(GameSession session, IPacketReader packet)
+    {
+        string title = packet.ReadUnicodeString();
+        string body = packet.ReadUnicodeString();
+
+        Player sender = session.Player;
+        Guild guild = session.Player.Guild;
```

Comment:
I would check for null since we can't trust the client also some kind of anti packet editing.

Reply:
Done, good spot :)